src/test/java/uk/ac/ebi/eva/commons/models/metadata/DatabaseTestConfiguration.java

```diff
@@ -15,8 +15,11 @@
  */
 package uk.ac.ebi.eva.commons.models.metadata;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
 
 /**
  * @author Oliver Gierke
```

Comment:
remove `@author` tags, in this PR or any another one, if this is already deployed to artifactory